all.sln

```diff
@@ -348,3 +347,5 @@ Global
 		SolutionGuid = {65220BF2-EAE1-4CB2-AA58-EBE80768CB40}
 	EndGlobalSection
 EndGlobal
+lobalSection
```

Comment:
Something looks wrong here. It should at least say `GlobalSection`, but are you sure you need this specific section of changes at all?

Reply:
Fixed